webcompat/models.py

```diff
@@ -8,15 +8,9 @@
 from sqlalchemy import Column
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy import Integer
-from sqlalchemy.orm import scoped_session
-from sqlalchemy.orm import sessionmaker
 from sqlalchemy import String
 from uuid import uuid4
-from webcompat import engine
-
-db_session = scoped_session(sessionmaker(autocommit=False,
-                                         autoflush=False,
-                                         bind=engine))
+from db import db_session, engine
```

Comment:
we use alphabetical order for import and one import per line.
see https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md#python